filebeat/harvester/reader/log.go

```diff
@@ -28,13 +28,13 @@ type logFileReader struct {
 }
 
 type LogFileReaderConfig struct {
-	CloseOlder         time.Duration
-	BackoffDuration    time.Duration
-	MaxBackoffDuration time.Duration
-	BackoffFactor      int
-	CloseRenamed       bool
-	CloseRemoved       bool
-	CloseEOF           bool
+	Backoff       time.Duration
+	MaxBackoff    time.Duration
+	BackoffFactor int
```

Comment:
It doesn't have the factor as it is just uses `*2`. We should do a more generic cleanup across all beats in an other PR.